clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -157,18 +166,29 @@ export default async function handler(
     const fidesRegionString = constructFidesRegionString(geolocation);
 
     if (fidesRegionString) {
+      // Check for a provided "fides_locale" query param or cookie. If present, use it as
+      // the user's preferred language, otherwise use the "accept-language" header
+      // provided by the browser. If all else fails, use the default.
+      const fidesLocale =
+        (req.query.fides_locale as string) || req.cookies?.fides_locale;
```

Comment:
do we set `fides_locale` anywhere across our ecosystem? Who uses this that we know of?

Reply:
I believe MediaVine is using it on a few of their sites that are language specific. Is there a specific concern you are considering here?

Reply:
nope, just curious about the wider implications across the ecosystem ✔️